### PR TITLE
Enable `throwSuggestions` RTL configuration 

### PR DESCRIPTION
**Pull request by amanmahajan7**
https://testing-library.com/docs/dom-testing-library/api-configuration/#throwsuggestions-experimental

### CODE REVIEW

File: test/setup.ts [hunk 1]
```diff
@@ -1,4 +1,7 @@
 import { act } from 'react-dom/test-utils';
+import { configure } from '@testing-library/react';
+
+configure({ throwSuggestions: true });
```
Comment (nstepien) [inline test/setup.ts:4]: >Duration  67.79s (transform 1.28s, setup 7.69s, collect 7.75s, tests 29.41s, environment 12.71s, prepare 2.69s)

The tests take over 1 minute now, I think it's fine to try it from time to time to improve tests, but I'm 👎 on keeping this enabled.

File: test/column/renderEditCell.test.tsx [hunk 2]
```diff
@@ -16,25 +16,25 @@ describe('Editor', () => {
   it('should open editor on double click', async () => {
     render(<EditorTest />);
     await userEvent.click(getCellsAtRowIndex(0)[0]);
-    expect(screen.queryByLabelText('col1-editor')).not.toBeInTheDocument();
+    expect(screen.queryByRole('spinbutton', { name: 'col1-editor' })).not.toBeInTheDocument();
```
Comment (nstepien) [inline test/column/renderEditCell.test.tsx:19]: What element are we actually querying here? I don't really understand what a spinbutton is

Comment (amanmahajan7) [reply test/column/renderEditCell.test.tsx:19]: `input type=number`

File: test/setup.ts (hunk 1 above)
Comment (amanmahajan7) [reply test/setup.ts]: How long does it take without this change?

Comment (nstepien) [reply test/setup.ts]: NVM I thought the GH CI was faster than that

Comment (nstepien) [reply test/setup.ts]: We can add it back

File: test/sorting.test.tsx [hunk 3]
```diff
@@ -35,9 +35,9 @@ function setup() {
 }
 
 function testSortColumns(expectedValue: readonly SortColumn[]) {
-  expect(JSON.parse(screen.getByTestId('sortColumnsValue').textContent!)).toStrictEqual(
-    expectedValue
-  );
+  expect(
+    JSON.parse(screen.getByTestId('sortColumnsValue', { suggest: false }).textContent!)
+  ).toStrictEqual(expectedValue);
```
Comment (nstepien) [inline test/sorting.test.tsx:40]: Can we use `toHaveTextContent`?

**Pull request merged**